.circleci/config.yml

```diff
@@ -0,0 +1,131 @@
+version: 2
+
+aliases:
+  - &test-base
+    working_directory: /tmp/drafter
+    environment:
+      CXXFLAGS: '-Werror -Wno-error=unused-function'
+
+    steps:
+      - checkout
+      - run: git submodule update --init --recursive
+      - run:
+          name: Build
+          command: |
+            ./configure
+            make
+      - run:
+          name: Test
+          command: make test
+
+  - &test-clang-base
+    <<: *test-base
+    environment:
+      CXXFLAGS: '-Werror -Wno-error=unused-function -Wno-error=pessimizing-move -Wno-error=missing-braces'
```

Comment:
Clang (both on macOS and Linux) produce further warnings, we should probably fix `pessimizing-move` although the `missing-braces` warning isn't really worth doing anything about and seems later versions of Clang remove this particular warning.